graphql/task_logs_resolver.go

```diff
@@ -11,13 +11,20 @@ import (
 	"github.com/evergreen-ci/evergreen/apimodels"
 	"github.com/evergreen-ci/evergreen/model"
 	"github.com/evergreen-ci/evergreen/model/event"
+	"github.com/evergreen-ci/evergreen/model/task"
 	restModel "github.com/evergreen-ci/evergreen/rest/model"
 	"github.com/evergreen-ci/utility"
 )
 
 func (r *taskLogsResolver) AgentLogs(ctx context.Context, obj *TaskLogs) ([]*apimodels.LogMessage, error) {
 	const logMessageCount = 100
-
+	task, taskErr := task.FindOneIdAndExecutionWithDisplayStatus(obj.TaskID, &obj.Execution)
```

Comment:
We don't actually check DisplayStatus so we can avoid the aggregation and just use [`task.FindOneIdAndExecution`](https://github.com/evergreen-ci/evergreen/blob/c082c88e7620c00df81d5e4d528fbdd144d27312/model/task/db.go#L1233)